etc/kayobe/environments/ci-aio/stackhpc-ci.yml

```diff
@@ -5,8 +5,9 @@
 # Docker namespace to use for Kolla images. Default is 'kolla'.
 kolla_docker_namespace: stackhpc-dev
 
-# Do not enable central logging if using Rocky Linux 9. It does not have elasticsearch
-kolla_enable_central_logging: "{{ os_release != '9' }}"
+# Rocky Linux 9 does not have elasticsearch, use opensearch instead.
+kolla_enable_elasticsearch: "{{ os_release != '9' }}"
```

Comment:
Let's go back to central logging for !RL9, since it enables kibana too.

Reply:
s/kibana/opensearch-dashboards/

Reply:
basically the default for Rocky9 on Yoga (upstream) is to use OpenSearch stack (and use Elasticsearch everywhere else) - so kolla_enable_central_logging: true should be enough

Reply:
Actually we have kolla_enable_central_logging=true in kolla.yml. Let's make these changes there, so that they apply everywhere.

We also need to make sure that there's a sensible way for users to switch to opensearch on CS8/RL8.

Reply:
> basically the default for Rocky9 on Yoga (upstream) is to use OpenSearch stack (and use Elasticsearch everywhere else) - so kolla_enable_central_logging: true should be enough

I'm not sure that's quite true:

```
enable_central_logging: "no"
enable_elasticsearch: "{{ 'yes' if not enable_opensearch | bool and (enable_central_logging | bool or enable_osprofiler | bool or enable_skydive | bool or enable_monasca | bool or (enable_cloudkitty | bool and cloudkitty_storage_backend == 'elasticsearch')) else 'no' }}"
enable_kibana: "{{ 'yes' if (enable_central_logging | bool and not enable_opensearch | bool) or enable_monasca | bool else 'no' }}"
enable_opensearch: "{{ kolla_base_distro == 'rocky' and enable_central_logging | bool }}"
```

So if you enable central logging on RL9 you get elastic and opensearch. Maybe this needs to be fixed upstream though?

Reply:
Hold on, I missed the `not enable_opensearch | bool` bit. I think @mnasiadka is correct then.

Reply:
I've just removed the statement disabling it for rocky, which if I understand this correctly should now end up with central logging being enabled for all AIOs & Multinodes, with Rocky 9s using opensearch/opensearch dashboards rather than elasticsearch/kibana